vignettes/EnrichmentBrowser.Rmd

```diff
@@ -0,0 +1,989 @@
+---
+title: "Seamless navigation through combined results of set- & network-based enrichment analysis"
+author:
+- name: Ludwig Geistlinger
+  affiliation: Center for Computational Biomedicine, Harvard Medical School
+abstract: >
+  The package `r Biocpkg("EnrichmentBrowser")` implements an analysis pipeline for high-throughput gene
+  expression data as measured with microarrays and RNA-seq. In a
+  workflow-like manner, the package brings together a selection of
+  established Bioconductor packages for gene expression data analysis.
+  It integrates a wide range of gene set and network enrichment analysis
+  methods and facilitates combination and exploration of results across
+  methods.
+
+package: EnrichmentBrowser
+output: 
+    BiocStyle::html_document:
+      toc_float: true
+vignette: >
+  %VignetteIndexEntry{Seamless navigation through combined results of set- & network-based enrichment analysis}
+  %VignetteEngine{knitr::rmarkdown}
+  %VignetteEncoding{UTF-8}  
+  
+bibliography:
+- refs.bib
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,eval = T,
+  comment = "#>",
+  dev = "jpeg"
+)
+```
+
+Report issues on
+<https://github.com/lgeistlinger/EnrichmentBrowser/issues>
+
+# Introduction
+
+The package `r Biocpkg("EnrichmentBrowser")` implements essential functionality for the enrichment analysis of gene expression data. The analysis combines the advantages of set-based and network-based enrichment analysis to derive
+high-confidence gene sets and biological pathways that are differentially regulated in the expression data under investigation.Besides, the package facilitates the visualization and exploration of such sets and pathways.
+The following instructions will guide you through an end-to-end
+expression data analysis workflow including:
+
+1.  Preparing the data
+
+2.  Preprocessing of the data
+
+3.  Differential expression (DE) analysis
+
+4.  Defining gene sets of interest
+
+5.  Executing individual enrichment methods
+
+6.  Combining the results of different methods
+
+7.  Visualize and explore the results
+
+All of these steps are modular, i.e. each step can be executed
+individually and fine-tuned with several parameters. In case you are
+interested in a particular step, you can directly move on to the
+respective section. For example, if you have differential expression
+already calculated for each gene, and your are now interested whether
+certain gene functions are enriched for differential expression, section
+*Set-based enrichment analysis* would be the one you should go for. The
+last section *Putting it all together* also demonstrates how to wrap the
```

Comment:
`Set-based enrichment analysis` and `Putting it all together` could be links.

Reply:
Removed the words and added reference.